src/pkg/utils/io.go

```diff
@@ -462,7 +462,9 @@ func CreateReproducibleTarballFromDir(dirPath, dirPrefix, tarballPath string) er
 		if err != nil {
 			return fmt.Errorf("error getting relative path: %w", err)
 		}
-		header.Name = filepath.Join(dirPrefix, name)
+		name = filepath.Join(dirPrefix, name)
+		name = filepath.ToSlash(name)
+		header.Name = name
```

Comment:
Can we make this a path? I would like it to go into a v0.32.3

Reply:
need more context